packages/p2p/src/components/page-return/page-return.tsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import classNames from 'classnames';
 import { Icon, Text } from '@deriv/components';
-import PropTypes from 'prop-types';
-import './page-return.scss';
 
-const PageReturn = ({ className, onClick, page_title }) => {
+type PageReturnProps = {
+    className?: string;
+    onClick: () => void;
+    page_title: string;
+};
+
+const PageReturn: React.FC<PageReturnProps> = ({ className = '', onClick, page_title }) => {
```

Comment:
AFAIK we don't use `React.FC` in our project (I don't like this but it was recommended not using this)
```suggestion
const PageReturn = ({ className = '', onClick, page_title }: TPageReturn) => {
```